hrms/hr/doctype/employee_onboarding_template/employee_onboarding_template.json

```diff
@@ -13,6 +13,39 @@
  "editable_grid": 1, 
  "engine": "InnoDB", 
  "fields": [
+  {
+   "allow_bulk_edit": 0, 
+   "allow_in_quick_entry": 0, 
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "fieldname": "title", 
+   "fieldtype": "Data", 
+   "hidden": 0, 
+   "ignore_user_permissions": 0, 
+   "ignore_xss_filter": 0, 
+   "in_filter": 0, 
+   "in_global_search": 0, 
+   "in_list_view": 1, 
+   "in_standard_filter": 0, 
+   "label": "Title", 
```

Comment:
Edit the doctype and set these fields as title fields in the doctype config
https://frappeframework.com/docs/user/en/basics/doctypes/form_&_view_settings#title-field